src/services/user-config-service.ts

```diff
@@ -34,15 +34,15 @@ export const updateRepoConfig = async (
 		try {
 			const repoSyncState = await RepoSyncState.findByRepoId(subscription, repositoryId);
 
-			if (!repoSyncState){
+			if (!repoSyncState) {
 				logger.error({
 					githubInstallationId,
 					repositoryId
 				}, "could not find RepoSyncState for repo");
 				return;
 			}
 
-			await updateRepoConfigFromGitHub(repoSyncState, githubInstallationId);
+			await updateRepoConfigsFromGitHub([repoSyncState], githubInstallationId);
```

Comment:
Switched to the new function so that the new function is covered by the existing tests.